packages/zero/tool/create-canary.js

```diff
@@ -99,7 +99,7 @@ function bumpCanaryVersion(version) {
 
 // To do a maintenance/cherry-pick release:
 // - create a maintenance release from tag you want to patch, like
-//   `maint/zero/vX.Y`
+//   `main/zero/vX.Y`
```

Comment:
<img width="535" alt="CleanShot 2025-03-20 at 04 49 57@2x" src="https://github.com/user-attachments/assets/d3ce7e5c-f3ef-4796-89ed-48f69ec8f51a" />

It was right as-is. "maint" is short for "maintenance".

Reply:
hehe, I'll undo